### PR TITLE
cantata: update to 2.3.0.

### PR DESCRIPTION
**Pull request by Hoshpak**

### CODE REVIEW

File: srcpkgs/cantata/template [hunk 1]
```diff
@@ -16,4 +16,4 @@ maintainer="LockeAnarchist <emanuel@openmailbox.org>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/cantata/template]: please use SPDX indetifier.

Comment (cr6git) [reply srcpkgs/cantata/template]: `GPL-3.0-or-later`

**Comment by cr6git** [comment]
@Hoshpak why you closed your PR ?

**Comment by Hoshpak** [comment]
@cr6git I'm fed up with that stupid licensing stuff and don't want to invest any more time into amending the PRs.

**Pull request closed without merging**